src/main/kotlin/no/digdir/fdk/searchservice/mapper/DatasetMapper.kt

```diff
@@ -0,0 +1,23 @@
+package no.digdir.fdk.searchservice.mapper
+
+import no.digdir.fdk.searchservice.model.*
+
+fun Dataset.toSearchObject() =
+    SearchObject(
+        id = id,
+        uri = "",
+        accessRights = accessRights,
+        catalog = null,
+        dataTheme = theme,
+        description = description,
+        fdkFormat = null,
+        harvest = null,
+        isOpenData = isOpenData,
+        keyword = keyword,
+        losTheme = losTheme,
+        organization = publisher?.toOrganization(),
```

Comment:
Skal ikke være nødvendig
```suggestion
        organization = publisher,
```

Reply:
ja, men er ikke nødvendig

Reply:
Når jeg fjerner det:

![Screenshot 2024-02-19 at 17 01 39](https://github.com/Informasjonsforvaltning/fdk-search-service/assets/50879748/9bdc99a8-2435-41a9-97fd-9c89325503ef)

Reply:
Fjern dataklassen Publisher helt, du trenger den ikke.

Reply:
Det er i sammenheng med kommentaren jeg hadde på Organization, om å fjerne title og i stedet bruke name og prefLabel. Da vil du kunne lese alle org-felt som Organization og har aldri behov for å endre den fra Publisher